QRCoder/QRCodeGenerator.cs

```diff
@@ -866,18 +874,13 @@ private static BitArray PlainTextToBinaryAlphanumeric(string plainText)
             return codeText;
         }
 
-        /// <summary>
-        /// Returns a string that contains the original string, with characters that cannot be encoded by a
-        /// specified encoding (default of ISO-8859-2) with a replacement character.
-        /// </summary>
-        private static string ConvertToIso8859(string value, string Iso = "ISO-8859-2")
-        {
-            Encoding iso = Encoding.GetEncoding(Iso);
-            Encoding utf8 = Encoding.UTF8;
-            byte[] utfBytes = utf8.GetBytes(value);
-            byte[] isoBytes = Encoding.Convert(utf8, iso, utfBytes);
-            return iso.GetString(isoBytes);
-        }
```

Comment:
Every time this method was called:
1. The string was converted from UTF-16 to UTF-8
2. The string was converted from UTF-8 to the target encoding
3. The string was converted from the target encoding back to UTF-16

Ultimately it did not do anything besides remove characters from the string that could not be represented in the target encoding